### ISSUE TITLE
"possibility of split size is too large" in version 1.2.0.4

### ISSUE THREAD

**Issue by ShangKent** [lltcggie/waifu2x-caffe#205]
The same as bug like:[https://github.com/lltcggie/waifu2x-caffe/issues/202](url).
![error1](https://user-images.githubusercontent.com/62099673/92851462-2bd91800-f420-11ea-9883-aba7896945a5.png)
![error2](https://user-images.githubusercontent.com/62099673/92851480-2f6c9f00-f420-11ea-80b7-703303447c78.png)

**Comment by fengweitai** [comment]
Ah!!Me too

**Comment by ShangKent** [comment]
> Did you ever figure it out?

No, I just know the last version that I can use is 1.2.0.2, CUDA version is 10.0 and cudnn version is 7.3.1.
Oh, my GPU is GTX 965m.

**Comment by JENYOK777** [comment]
Try to change model. I have same problem due to out of VRAM.
